Application/components/Messaging/MessagingInterface.tsx

```diff
@@ -54,21 +51,18 @@ export function MessagingInterface({ senderUsername, senderID, receiverIDs, priv
   let messageEnd: HTMLDivElement | null = null;
   const inputBoxRef = useRef(null);
   const messageEndRef = React.useRef<HTMLDivElement>(null);; // Updated to use useRef
-  const [myAblyClientID, setMyAblyClientID] = useState('');
-
   const [messageText, setMessageText] = useState(""); // messageText is bound to a textarea element where messages can be typed.
   const [receivedMessages, setReceivedMessages] = useState<DisplayedMessageProps[]>([]); // receivedMessages stores the on-screen chat history.
   const memberIDs = [senderID, ...receiverIDs] // / To allow for group chats, we are creating an array that contains the sender and the receivers.
-  memberIDs.sort(); // CRITICAL: Sorts in-place. We need to sort the key to counteract the swapping mechanism where sender and receiver becomes flipped.
   // Retrieving the chat history and update function from the context
   const { chatHistory, updateChatHistory } = useChat();
   const messageTextIsEmpty = messageText.trim().length === 0; // messageTextIsEmpty is used to disable the send button when the textarea is empty.
 
   // useChannel is a react-hook API for subscribing to messages from an Ably channel.
   // You provide it with a channel name and a callback to be invoked whenever a message is received.
   // Both the channel instance and the Ably JavaScript SDK instance are returned from useChannel.
-  const rawChannelKey = `chat:${memberIDs.join(",")}`;
-  const channelKey = generateHash(rawChannelKey); // Generating a SHA-256 hash of a channel to compress it and also enforce security, privacy and uniqueness :D
+  
+  const channelKey = generateHash(memberIDs);
```

Comment:
We did have a talk about how the channelKeys swaps as soon as a member gets deleted - good one to append towards this.